libsignal-service/src/cipher.rs

```diff
@@ -78,12 +83,27 @@ where
             let plaintext = self.decrypt(&envelope).await?;
             let message =
                 crate::proto::DataMessage::decode(plaintext.data.as_slice())?;
-            Ok(Some(Content::from_body(message, plaintext.metadata)))
+            let content = Content::from_body(message, plaintext.metadata);
+            Ok(Some(content))
```

Comment:
I don't see the relevance of this?

Reply:
`Content::from_proto` now returns a `Result` so we can precisely see in the error logs when a new type of message comes in that we don't support (we returned `None` before which was IMO misleading). Let me double check if there's a better way to do this.